Adafruit_GFX.cpp

```diff
@@ -71,596 +71,680 @@ POSSIBILITY OF SUCH DAMAGE.
 #endif
 
 Adafruit_GFX::Adafruit_GFX(int16_t w, int16_t h):
-  WIDTH(w), HEIGHT(h)
```

Comment:
The diff here is strange with spacing differences from the previous one along with many other spots, is this intentional?